src/app/zap-templates/zcl/data-model/chip/access-control-cluster.xml

```diff
@@ -33,6 +33,14 @@ limitations under the License.
     <item name="Group" value="0x03"/>
   </enum>
 
+  <enum name="AccessRestrictionTypeEnum" type="enum8">
```

Comment:
I am looking to compare this to the spec, updated master spec right now and I don't seem to find this type. 
`rg 'Access.*Restriction'` comes up as empty.

Where should I be looking at? Is there a spec change in flight?

Reply:
If this is a in-flight/in-review spec PR, please provide a link in the PR summary for reviewers to have.

Reply:
summary updated.